rails-tasklist/app/views/people/list.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+  <h1><%= @person.name %></h1>
+    <table class="table table-hover">
+      <tr >
+        <th>
+        </th>
+        <th>
+        </th>
+        <th>
+        </th>
+        <th>
+        </th>
+        <th>
+        </th>
+        <th>
+          <strong>task</strong>
+        </th>
+      </tr>
+      <% @person.tasks.each do |task| %>
+        <tr>
+          <td>
+            <%= link_to "show", "/tasks/#{task.id}", class: "btn btn-default" %>
+          </td>
+          <td>
+            <%= button_to "delete", "/tasks/#{task.id}", method: :delete, class: "btn btn-default", data: { confirm: 'Are you sure?' } %>
+            <!-- onclick="return confirm('Are you sure?')" -->
+          </td>
+          <td>
+            <!-- <%= button_to "complete", "/tasks/#{task.id}/complete", method: :patch, class: "btn btn-default" %> -->
+          </td>
+          <td>
+            <%= button_to "edit", "/tasks/#{task.id}/edit", method: :get, class: "btn btn-default" %>
+          </td>
+          <td>
+            <div><%= task.person.name %></div>
+          </td>
+         <td>
+          	<div><%= task.name %></div>
+         </td>
+         <td>
+           <div>
+             <% if task.completed_at.nil? %>
+              <%= button_to "complete", "/tasks/#{task.id}/complete", method: :patch, class: "btn btn-default" %>
+                <!-- <img class="nocheckmark" src="/assets/white.png" alt="no checkmark"/> -->
+              <% else %>
+                <img class="checkmark" src="/assets/checkmark3.png" alt="checkmark"/>
+              <% end %>
+            </div>
+          	<!-- <div><% if task.completed_at.class == NilClass %><%= "in progress :)" %><% end %><%= task.completed_at %></div> -->
```

Comment:
I would remove the code that you don't need when you submit. Also, look out for indentation in this similar block.